domains/org.eclipse.lyo.tools.domainmodels/oslcDomainSpecifications.xml

```diff
@@ -82,18 +88,21 @@
     <resourceProperties id="_posD4ZUdEeq-KoPaR9_Cfg" name="label" description="May be used to provide a human-readable version of a resource's name." occurs="zeroOrOne" valueType="XMLLiteral">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='label']"/>
     </resourceProperties>
+    <resourceProperties id="_uuAX4JUdEeq-KoPaR9_Cfg" name="label" description="May be used to provide a human-readable version of a resource's name." occurs="zeroOrOne" valueType="String">
+      <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='label']"/>
+    </resourceProperties>
     <resourceProperties id="_poyxkZUdEeq-KoPaR9_Cfg" name="subClassOf" description="used to state that all the instances of one class are instances of another" valueType="Resource" range="_poyKgJUdEeq-KoPaR9_Cfg">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='subClassOf']"/>
     </resourceProperties>
     <resourceProperties id="_posD4JUdEeq-KoPaR9_Cfg" name="comment" description="May be used to provide a human-readable description of a resource." valueType="XMLLiteral">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='comment']"/>
     </resourceProperties>
+    <resourceProperties id="_rZmPgFv1EeyHYL9SbPncDg" name="comment" description="May be used to provide a human-readable description of a resource." valueType="String">
+      <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='comment']"/>
+    </resourceProperties>
     <resourceProperties id="_poyxkpUdEeq-KoPaR9_Cfg" name="seeAlso" description="Used to indicate a resource that might provide additional information about the subject resource." occurs="zeroOrOne" valueType="URI">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='seeAlso']"/>
     </resourceProperties>
-    <resourceProperties id="_uuAX4JUdEeq-KoPaR9_Cfg" name="label" valueType="String">

```

Comment:
Note that this property was not deleted, It still exists (line 91), including a description of the property.